actions/build-push-docker-manifest/README.md

```diff
@@ -0,0 +1,134 @@
+# build-push-docker-manifest
+
+## Example usage
+
+Reusable composite workflow to create docker manifest for multi-platform. Use
+this after the `build-push-docker` composite workflow to create and push the
+manifest which will be an index of the image(s) from the `build-push-docker`
```

Comment:
Might want to hyperlink `build-push-docker`?

Reply:
0747943fdac188bf3df37dd1221dda6636d19a67